src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglx_utils.py

```diff
@@ -105,6 +107,7 @@ def get_device_metadata(meta) -> dict:
             "21": "NP2.0(1-shank)",
             "24": "NP2.0(4-shank)",
             "1030": "NP1.0 NHP",
+            "2013": "NP2.0(4-shank)",
```

Comment:
Is there any example data on GIN we can use to test this one?

Reply:
Not sure.

The soluton that we went for here of building a local table is bad. We should do this from probeinterface anyway.

Reply:
It would be better at ProbeInterface level, yes

Reply:
Would be nice to have test data for this shank logic, just like how I added the multi probe data to demonstrate the new device adding logic

Reply:
Let me remove this line as it is totally incidental.

Reply:
Ok, moved this to another PR.

https://github.com/catalystneuro/neuroconv/pull/990

Reply:
I'm saying it would be fine to include it this line if you also add multishank specific tests on multishank example data, to prove your improvements to the shank metadata and leave of record of what we expect that metadata to look like

Right now it only proves that the single-shank data doesn't include the unitary shank details per probe

Basically asking for tests just like what we did for the multi probe case, but for multi shank instead

Then we can say and prove SpikeGLX metadata is in a good place

Reply:
We already have tests for multishank neuropixel 2.0 here. It was a good suggestion as I added tests for the new added attributes.

https://github.com/catalystneuro/neuroconv/blob/5ddffc6ff0eb24b12f594cd323bc9d7a71c197b5/tests/test_on_data/test_metadata/test_spikeglx_metadata.py#L27-L51